validator/accounts/v2/export.go

```diff
@@ -25,7 +27,7 @@ func ExportAccount(cliCtx *cli.Context) error {
 	// Read a wallet's directory from user input.
 	walletDir, err := inputWalletDir(cliCtx)
 	if errors.Is(err, ErrNoWalletFound) {
-		return errors.New("no wallet found, create a new one with ./prysm.sh validator wallet-v2 create")
+		log.Fatal("no wallet found, create a new one with ./prysm.sh validator wallet-v2 create")
```

Comment:
Why not return error as we discussed?

Reply:
Was waiting for the wrapper you mentioned to be implemented, maybe it can be done for all of accounts-v2 in a PR?